docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -208,7 +218,7 @@ The following EBNF grammar summarizes the structure of commits and ledgers:
    Ledger   ::= Commit*
 
 A DA ledger thus represents the full history of all actions taken by
-parties. [#ledger-vs-journal]_ Since the ledger is a sequence (of dependent actions), it induces an
+parties.\ [#ledger-vs-journal]_ Since the ledger is a sequence (of dependent actions), it induces an
```

Comment:
What's this change? The footnote seemed to work fine before, or am I missing something

Reply:
There was a space between the `.` and the footnote mark. With the `\`, the footnote mark sits directly after the full stop.